packages/components/src/utils/OSfunctions.tsx

```diff
@@ -110,3 +110,29 @@ export function useExternalLink(): (
     }
   }
 }
+
+/**
+ * Returns whether or not the device has a home button
+ */
+export const hasHomeButton = () => {
+  const { height, width } = Dimensions.get('window')
+
+  if (Platform.OS !== 'ios') {
```

Comment:
```suggestion
  if (!isIOS()) {
```

Reply:
Nitpicky: could also move above the Dimensions call since it just gets dropped on the floor